app/models/event_type_filter.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 class EventTypeFilter < ApplicationRecord
-  belongs_to :event_type
-  validates_associated :event_type
+  belongs_to :event_type, :optional => true
```

Comment:
The same as for `AppFilter` - we don't need to make these associations optional.